autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -239,7 +242,7 @@ def train_single(self, X_train, y_train, X_val, y_val, model, kfolds=None, k_fol
             model.fit(X_train=X_train, y_train=y_train, X_val=X_val, y_val=y_val, time_limit=time_limit, **model_fit_kwargs)
         return model
 
-    def train_and_save(self, X_train, y_train, X_val, y_val, model: AbstractModel, stack_name='core', kfolds=None, k_fold_start=0, k_fold_end=None, n_repeats=None, n_repeat_start=0, level=0, time_limit=None):
+    def _train_and_save(self, X_train, y_train, X_val, y_val, model: AbstractModel, stack_name='core', kfolds=None, k_fold_start=0, k_fold_end=None, n_repeats=None, n_repeat_start=0, level=0, time_limit=None):
```

Comment:
documentation should describe how reference to this model will be stored in this `Trainer` object.

Reply:
added